modpack_format/modpack_example.modip.json

```diff
@@ -0,0 +1,35 @@
+{
+  "formatType": "modipModpack",
+  "formatVersion": "1.0.0",
+  "id": "modpack-example",
+  "name": "Modpack Example",
+  "summary": "Summary of the pack",
+  "description": "Detailed description of the modpack. <b>HTML?</b>",
+  
+  "updates": "https://api.example.com/pack/modpack-example.modip.json",
+
+  "dependencies": [
+    {
+      "id": "minecraft",
+      "version": "1.16.4"
+    },
+    {
+      "id": "fabric-loader",
+      "version": "0.9.0"
+    },
+    {
+      "id": "mymod",
+      "version": "1.0.0",
+      "updates": "https://api.example.com/mod/mymod.modip.json",
+      "files": [
```

Comment:
Is it sensible to be able to update an individual file in a modpack?

Reply:
Maybe? But it may not be implemented by everyone, and if it moves to a generic files instead of dependency array this will go away anyway.